### PR TITLE
Export plots to .svg and toggleable exporting

### PR DESCRIPTION
**Pull request by AndreasPB**
* Exports all plots to .svg
* Add `EXPORT_PLOTS` const to be able to run files without overriding the exports

### CODE REVIEW

**Comment by AndreasPB** [comment]
Solves #35

File: src/plots/vote_plot.py [hunk 1]
```diff
@@ -74,3 +81,8 @@ def split_by_ratings(data: pd.DataFrame) -> list[list]:
 
 fig.tight_layout()
 plt.show()
+if EXPORT_PLOTS:
+    plt.savefig(
+        "../../report/img/rating_vote_heatmap_excluding_no_votes.svg",
```
Comment (Pankai222) [inline src/plots/vote_plot.py:86]: There seems to be something wrong with the heatmap svg generated. It won't render here, in my IDE previewer or using an online SVG previewer

Comment (AndreasPB) [reply src/plots/vote_plot.py]: FIXED! 🔧

**Review by Pankai222 — APPROVED** [review]

**Pull request merged**